Source/WebCore/platform/graphics/texmap/coordinated/CoordinatedGraphicsLayer.cpp

```diff
@@ -1375,8 +1387,13 @@ void CoordinatedGraphicsLayer::computeTransformedVisibleRect()
 
     m_shouldUpdateVisibleRect = false;
     TransformationMatrix currentTransform = transform();
-    if (m_movingVisibleRect)
+        TransformationMatrix futureTransform = currentTransform;
+    if (m_movingVisibleRect) {
         client().getCurrentTransform(this, currentTransform);
+	        Nicosia::Animation::ApplicationResult futureApplicationResults;
+        m_animations.applyKeepingInternalState(futureApplicationResults, MonotonicTime::now() + 300_ms);
```

Comment:
300 ms looks like an arbitrary hardcoded value - why 300? Won't this break other animations? What if the animation was faster (or slower) ?

Reply:
On this , we received comment from upstream team like below.
>>The value how far we need to look in the future in case of animation highly depends on the speed of the device. In case of PC it is only 50ms, in case RPi it was 300ms. Probably it would be better to make it not hardcoded but configurable (e.g.: env var).